src/font/text.cpp

```diff
@@ -618,6 +620,21 @@ static void unpremultiply(uint8_t & value, const unsigned div) {
 	value = std::min(255u, temp);
 }
 
+static inline void overblend(uint8_t& a, uint8_t& r, uint8_t& g, uint8_t& b)
+{
+	if(a == 255) {
+		return;
+	}
+
+	unsigned tmp_r = 1 + r * 255;
+	r = (tmp_r + (tmp_r >> 8)) >> 8;
+	unsigned tmp_g = 1 + g * 255;
+	g = (tmp_g + (tmp_g >> 8)) >> 8;
+	unsigned tmp_b = 1 + b * 255;
+	b = (tmp_b + (tmp_b >> 8)) >> 8;
```

Comment:
AFAICS the non-alpha channels always end up with the same values at the end as they have at the start.

Plotting it with Python:
```
>>> for r in range(0, 256):
...     tmp_r = 1 + r * 255
...     print (r - ((tmp_r + (tmp_r >> 8)) >> 8))
... 
0
lots more zeros
```

Reply:
You're not wrong. The concept itself is dumb since obviously you can't mix (r, g, b) and (r, g, b) and expect something different in return. I'm sleepy.